care/facility/models/daily_round.py

```diff
@@ -462,3 +515,14 @@ def has_object_write_permission(self, request):
         ):
             return False
         return self.has_object_read_permission(request)
+
+    def has_object_asset_read_permission(self, request):
+        return False
+
+    def has_object_asset_write_permission(self, request):
+        consultation = PatientConsultation.objects.get(
+            external_id=request.parser_context["kwargs"]["consultation_external_id"]
+        )
+        return consultation.current_bed.bed.bed_set.filter(
```

Comment:
Lets use the AssetBed Class to query, using set can be confusing for the most part.